extensions/amp-story/1.0/amp-story-render-service.js

```diff
@@ -15,8 +15,17 @@
  */
 
 import {CommonSignals} from '../../../src/common-signals';
+import {Services} from '../../../src/services';
 import {whenUpgradedToCustomElement} from '../../../src/dom';
 
+/**
+ * Maximum milliseconds to wait for service to load before logging a warning.
```

Comment:
Plz either log a warning or remove the comment. I'm not sure we do need a warning though.